### PR TITLE
make travis directories dynamic

### PR DESCRIPTION
**Pull request by bobholt**
This dynamically sets the directories used in the `check_stability.py` script to be based on the user running the Travis job.

This allows anyone to set up Travis builds on their fork of the web platform tests without modifying `check_stability.py`.

This opts to pass the user value to each method rather than setting a global value. Happy to change if a global value is preferred.

### CODE REVIEW

**Comment by jgraham** [comment]
You might need to rebase onto latest master for this to work.

**Comment by jgraham** [comment]
Seems OK in principle, but it's pretty noisy. Maybe we should make the base path a global which we write to once?

**Comment by bobholt** [comment]
@jgraham Agree. I'll make an update.

**Comment by bobholt** [comment]
Latest push makes that user variable global. Will go further and make that whole path global now.

**Review by jgraham — APPROVED** [review]
Can you just demonstrate this works by pushing a no-op test change, and then removing it again once the checker reports some results?

File: check_stability.py [hunk 1]
```diff
@@ -606,9 +606,8 @@ def main():
     parser = get_parser()
     args = parser.parse_args()
 
-    user_path = os.path.join(os.path.abspath(os.curdir), args.user)
-    wpt_root = os.path.join(user_path, "web-platform-tests")
-    wptrunner_root = os.path.join(user_path, "wptrunner")
+    wpt_root = os.path.join(os.path.abspath(os.curdir))
+    wptrunner_root = os.path.join(wpt_root, "..", "wptrunner")
```
Comment (jgraham) [inline check_stability.py]: I think a normalized path here would be nice.

**Review by jgraham — CHANGES_REQUESTED** [review]

File: check_stability.py (hunk 1 above)
Comment (bobholt) [inline check_stability.py]: Just had that same thought and pushed another change - rebased because I was getting so many fixups.

**Comment by bobholt** [comment]
Please bear with me as I work through my python rust. If it's too noisy, I can close and reopen once the light is green on my fork.

**Review by jgraham — APPROVED** [review]

**Comment by jgraham** [comment]
I don't know what the problem reporting the travis status is, but the build passes and tests run, so I'm happy with this as soon as you remove the example test change.

**Comment by bobholt** [comment]
Yeah, that status was causing me great consternation yesterday. The test change has been removed.

**Pull request merged**